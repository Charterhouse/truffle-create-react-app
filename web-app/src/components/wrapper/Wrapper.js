import React from 'react'

export const Wrapper = ({ children }) =>
  <div style={{ padding: '20px' }}>
    {children}
  </div>
