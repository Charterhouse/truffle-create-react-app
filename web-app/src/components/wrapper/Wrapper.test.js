import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { Wrapper } from './Wrapper'

let exampleWrapper

beforeEach(() => {
  exampleWrapper = <Wrapper><p>Some Text</p></Wrapper>
})

it('renders without crashing', async () => {
  const div = document.createElement('div')
  await ReactDOM.render(exampleWrapper, div)
})

it('renders correctly', () => {
  const component = renderer
    .create(exampleWrapper)
  expect(component.toJSON()).toMatchSnapshot()
})
