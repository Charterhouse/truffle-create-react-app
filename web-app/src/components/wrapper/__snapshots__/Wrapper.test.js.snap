// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "padding": "20px",
    }
  }
>
  <p>
    Some Text
  </p>
</div>
`;
